### PR TITLE
Updated with what I needed to do to make it work

### PR DESCRIPTION
**Pull request by paulrobinson**
This is what I needed to do to make this work.

I removed the generated content from the .env file. Given that it contains secrets, we would probably want to store an instance of this file somewhere secure when the GitHub app is created for production. 

I also changed the requested permissions to allow write access to repository and organisation level projects. I'm not sure how this could have worked before, so worth sanity checking this change.

Finally, I expanded the docs to describe the steps I followed to make it work.

### CODE REVIEW

File: app.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ default_events:
 # - project_card
 # - project_column
 # - public
-- pull_request
+# - pull_request
```
Comment (maxandersen) [inline app.yml]: why did you need to remove this ? without this you can't put PR's on the boards.

**Comment by maxandersen** [comment]
did a forceful removal of .env which rewrote history so you'll need to rebase.

File: app.yml (hunk 1 above)
Comment (paulrobinson) [reply app.yml]: I was getting this when registering the bot:

image

Agreed, we do need to be notified about PRs. I planned to see what was causing this, but ran out of time on Friday.

**Comment by paulrobinson** [comment]
Rebased on here: https://github.com/paulrobinson/mighty-bot/tree/MAKE_WORK_2

I'll raise a PR when I have figured out what is going on with the PR events.
